### PR TITLE
Proxsdpdocs

### PR DESCRIPTION
**Pull request by guilhermebodin**
We would like to add ProxSDP in the solvers' list.

cc @joaquimg

### CODE REVIEW

File: docs/src/installation.md [hunk 1]
```diff
@@ -146,6 +147,15 @@ julia> using MosekTools
 julia> model = Model(with_optimizer(Mosek.Optimizer))
 ```
 
+### ProxSDP
+
+ProxSDP solves general SDP problems by means of a first order proximal algorithm
+based on the primal-dual hybrid gradient, also known as Chambolle-Pock method.
+The main advantage of ProxSDP over other state-of-the-art solvers is the ability 
+to exploit the low-rank property inherent to several SDP problems. ProxSDP 
+is a first order solver and has low accuracy. see the ProxSDP.jl
```
Comment (blegat) [inline docs/src/installation.md]: see -> See

File: docs/src/installation.md [hunk 2]
```diff
@@ -146,6 +147,15 @@ julia> using MosekTools
 julia> model = Model(with_optimizer(Mosek.Optimizer))
 ```
 
+### ProxSDP
+
+ProxSDP solves general SDP problems by means of a first order proximal algorithm
+based on the primal-dual hybrid gradient, also known as Chambolle-Pock method.
+The main advantage of ProxSDP over other state-of-the-art solvers is the ability 
+to exploit the low-rank property inherent to several SDP problems. ProxSDP 
+is a first order solver and has low accuracy. See the ProxSDP.jl
```
Comment (blegat) [inline docs/src/installation.md]: Add an hyperlink for ProxSDP.jl

File: docs/src/installation.md [hunk 3]
```diff
@@ -42,6 +42,7 @@ is often more complex. We list below the currently available solvers.
 | [Ipopt](https://projects.coin-or.org/Ipopt)                                    | [Ipopt.jl](https://github.com/JuliaOpt/Ipopt.jl)                                 | EPL     | LP, QP, NLP                        |
 | [MOSEK](http://www.mosek.com/)                                                 | [MosekTools.jl](https://github.com/JuliaOpt/MosekTools.jl)                       | Comm.   | LP, MILP, SOCP, MISOCP, SDP        |
 | [OSQP](https://osqp.org/)                                                      | [OSQP.jl](https://github.com/oxfordcontrol/OSQP.jl)                              | Apache  | LP, QP                             |
+| [ProxSDP](https://github.com/mariohsouto/ProxSDP.jl)                                | [ProxSDP.jl](https://github.com/mariohsouto/ProxSDP.jl)                     | MIT     | LP, SOCP, SDP                      |
```
Comment (mlubin) [inline docs/src/installation.md]: Markdown is very picky about the formatting of tables. It's hard to tell from the diff if the whitespace is correct. Did you try [building the docs](https://github.com/JuliaOpt/JuMP.jl/tree/master/docs#building-the-documentation) to confirm that the table build correctly?

Comment (guilhermebodin) [reply docs/src/installation.md]: I have tried previewing in vs code but building is indeed more guaranteed

image

**Pull request merged**